config_samples/ethereum/hoodi/hoodi_vaults_testnet_config.json

```diff
@@ -1,425 +1,394 @@
+    "0x16e2D8ABc9d0682A10e10Ba5300DF000757BE522": "LidoLocator",
+    "0x3508A952176b3c15387C97BE809eaffB1982176a": "AppProxyUpgradeable",
+    "0x35A4f9c9c2B1f81bDe7Eaa1f23b6465D3d741EEF": "Lido",
+    "0x7E99eE3C66636DE415D2d7C880938F2f40f94De4": "WstETH",
+    "0x2A1d51BF3aAA7A7D027C8f561e5f579876a17B0a": "EIP712StETH",
+    "0xCc820558B39ee15C7C45B59390B503b83fb499A8": "OssifiableProxy",
+    "0xd5F04A81ac472B2cB32073CE9dDABa6FaF022827": "StakingRouter",
+    "0x2F0303F20E0795E6CCd17BD5efE791A586f28E03": "DepositSecurityModule",
+    "0x9b108015fe433F173696Af3Aa0CF7CDb3E104258": "LidoExecutionLayerRewardsVault",
+    "0xfe56573178f1bcdf53F01A6E9977670dcBBD9186": "OssifiableProxy",
+    "0xD0a60e52837e045F4567193Cf8921191C486eCD5": "WithdrawalQueueERC721",
+    "0xfe7A58960Af333eAdeAeC39149F9d6A71dc3E668": "WithdrawalVault",
+    "0x9b5b78D1C9A3238bF24662067e34c57c83E8c354": "OssifiableProxy",
+    "0x0bF902fb783Fbf8af0bC011C76D2F7d318a50c74": "Accounting",
+    "0xb2c99cd38a2636a6281a849C8de938B3eF4A7C3D": "OssifiableProxy",
+    "0xBb77f95A29D1ccE1A54F6ED9144D48be38D43dbC": "Burner",
+    "0x6d1a9bBFF97f7565e9532FEB7b499982848E5e07": "MinFirstAllocationStrategy",
+    "0x6679090D92b08a2a686eF8614feECD8cDFE209db": "TriggerableWithdrawalsGateway",
+    "0xa5F5A9360275390fF9728262a29384399f38d2f0": "ValidatorExitDelayVerifier",
+    "0x4C9fFC325392090F789255b9948Ab1659b797964": "OssifiableProxy",
+    "0xAA80eE34a21D48Ef2C99A2403101E08e4Fd18726": "VaultHub",
+    "0xa5F55f3402beA2B14AE15Dae1b6811457D43581d": "OssifiableProxy",
+    "0xDf9d7ce12DD859fa929A3Ef01Bc3BecF78fD3755": "PredepositGuarantee",
+    "0x501e678182bB5dF3f733281521D3f3D1aDe69917": "OssifiableProxy",
+    "0x3C17C3feeaccdDA22Ed72e386c20E168f3CBa3c9": "OperatorGrid",
+    "0x67Fc99587B4Cd6FA16E26FF4782711f79055d7ad": "VaultFactory",
+    "0xb3e6a8B6A752d3bb905A1B3Ef12bbdeE77E8160e": "UpgradeableBeacon",
+    "0xdc79D1751D1435fEc9204c03ca3D64ceEB73A7df": "StakingVault",
+    "0x7b2B6EA1e53B2039a493cA587805183883Cb8B88": "Dashboard",
+    "0xbf95Cd394cC03cD03fEA62A435ac347314877f1d": "ValidatorConsolidationRequests",
+    "0xcb883B1bD0a41512b42D2dB267F2A2cd919FB216": "OssifiableProxy",
+    "0x6D799F4C92e8eE9CC0E33367Dd47990ed49a21AC": "AccountingOracle",
+    "0x32EC59a78abaca3f91527aeB2008925D5AaC1eFC": "HashConsensus",
+    "0x8664d394C2B3278F26A1B44B967aEf99707eeAB2": "OssifiableProxy",
+    "0x7E6d9C9C44417bf2EaF69685981646e9752D623A": "ValidatorsExitBusOracle",
+    "0x30308CD8844fb2DB3ec4D056F1d475a802DCA07c": "HashConsensus",
+    "0x53417BA942bC86492bAF46FAbA8769f246422388": "OracleReportSanityChecker",
+    "0x2a833402e3F46fFC1ecAb3598c599147a78731a9": "OracleDaemonConfig",
+    "0xf41491C79C30e8f4862d3F4A5b790171adB8e04A": "OssifiableProxy",
+    "0x4965052b7aa07375A261317Daa2823A159c36ef6": "LazyOracle",
+    "0x5cDbE1590c083b5A2A64427fAA63A7cfDB91FbB5": "AppProxyUpgradeable",
+    "0x95F00b016bB31b7182D96D25074684518246E42a": "NodeOperatorsRegistry",
```

Comment:
`NodeOperatorsRegistry.sol` and `ILidoLocator.sol` changed. In locator `oracleReportComponentsForLido()` ⇒ `oracleReportComponents()` need to check that is was not used anywhare in old code.

Reply:
Looks like it was used only in `Lido` and subject to change in V3, so all good!